amundsen_application/static/js/fixtures/mockRouter.ts

```diff
@@ -25,14 +25,14 @@ export function getMockRouterProps<P>(data: P, location: Partial<History.Locatio
       length: 2,
       action: 'POP',
       location: mockLocation,
-      push: () => {},
-      replace: null,
-      go: null,
-      goBack: null,
-      goForward: null,
-      block: null,
-      createHref: null,
-      listen: null,
+      push: jest.fn(),
```

Comment:
Is there any reason why we wouldn't want to create mock functions for `go`, `goBack`, etc?

Reply:
Mostly curious about the need. When creating this to use in tests, I took an objective approach to only add what we need. But the main idea is that I didn't expect the `jest` object to be in scope outside of a test file, and question if that should be the case and if we should continue to allow it. Even though this is imported into test files, as it stands its just a normal typescript file and would expect `jest` to be used/accessed only after it is imported into a test file. 

I am not opposed to this getting merged as is, but my preference is to not leave unused methods as null and assign mock functions in the test file itself. At some point I'd like to dig a bit deeper into whether or not `jest` should be able to be accessed outside of the tests, and if there's anything we'd want to do about it.

Reply:
I've reverted the unused functions to `null` and the used ones to `() => {}`, but what's the downside to just putting `go: (href) => {}` as a placeholder so if we need to test it later on we can.

Reply:
Oh I see. There's no downside that would be the end of the world, the short version is that I didn't add it because I didn't have a reason to add it. I considered it unneeded work to look up the signatures for all of these if we aren't using them. Additionally our unit test coverage will consider those empty methods uncovered, and they would remain that way because there's no reason to test them, so I also opted not to add code.